hooks/boot/12-clear_pulp_content.rb

```diff
@@ -0,0 +1,8 @@
+if pulp_enabled?
```

Comment:
With Pulp 2 removal planned and being worked on for 4.0, wouldn't it be better to just remove it from Katello and drop it?

Reply:
I think it could be kept until the removal is really complete? Having it split into separate files makes it trivially easy to drop.

Reply:
We're actively working on killing Pulp 2 in all modules. Resetting and clearing is IMHO a good place to start. Katello 4.0 is the big Pulp 2 removal version and we slightly less than a month (so the end of this sprint) before branching. I'm in favor of doing the work just once: remove first, then move over what's left.

Reply:
Maybe we hedge our bets, and go this route until we've actually removed Pulp 2. This keeps us flexible but easily able to rip this out closer to release. Despite the project being `Pulp` so much naming has had to be `pulpcore` that I do think using that for the Pulp 3 hook makes sense. Provides consistency.

Reply:
I thought about it more and I think this is the right approach until we know the Katello plugin is fully ready to drop Pulp 2 given their timeline has slipped due to various bugs found in the migrations and adaptation to Pulp 3 RPM content.

Reply:
Given Katello project will be having two more releases, I think the two options is the route to go.